### PR TITLE
Update bakta from 1.8.1 to 1.8.2

### PR DESCRIPTION
**Pull request by pimarin**

### CODE REVIEW

File: tools/bakta/macro.xml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 
 <macros>
-    <token name="@TOOL_VERSION@">1.8.1</token>
+    <token name="@TOOL_VERSION@">1.8.2</token>
     <token name="@COMPATIBLE_BAKTA_VERSION@">1.7</token>
-    <token name="@VERSION_SUFFIX@">1</token>
+    <token name="@VERSION_SUFFIX@">2</token>
```
Comment (bernt-matthias) [inline tools/bakta/macro.xml:5]: suffix should be reset to 0 for new versions

```suggestion
    <token name="@VERSION_SUFFIX@">0</token>
```

File: tools/bakta/bakta.xml [hunk 2]
```diff
@@ -343,7 +343,7 @@
             <output name="summary_txt" value="TEST_6/TEST_6.txt" lines_diff="4"/>
             <output name="annotation_plot" value="TEST_6/TEST_6_plot.svg" ftype="svg" compare="sim_size"/>
            <output name="annotation_json" value="TEST_6/TEST_6.json" lines_diff="6"/>
-           <output name="logfile" value="TEST_6/TEST_6.log" lines_diff="15"/>
+           <output name="logfile" value="TEST_6/TEST_6.log" lines_diff="20"/>
```
[the remark this answers is not in the document]
Comment (pimarin) [reply tools/bakta/bakta.xml:346]: oky I will change it

File: tools/bakta/bakta.xml [hunk 3]
```diff
@@ -233,7 +233,17 @@
            <output name="summary_txt" value="TEST_1/TEST_1.txt" lines_diff="4"/>
            <output name="annotation_plot" value="TEST_1/TEST_1_plot.svg" ftype="svg" compare="sim_size"/>
            <output name="annotation_json" value="TEST_1/TEST_1.json" lines_diff="6"/>
-           <output name="logfile" value="TEST_1/TEST_1.log" lines_diff="20"/>
+           <output name="logfile" value="TEST_1/TEST_1.log" ftype="txt" lines_diff="15">
```
Comment (bernt-matthias) [inline tools/bakta/bakta.xml:236]: The diff is still executed as long as `value` is defined:

```suggestion
           <output name="logfile" ftype="txt" lines_diff="15">
```
The `assert_content` may go to a macro if you like.

File: tools/bakta/bakta.xml [hunk 4]
```diff
@@ -232,8 +232,18 @@
            <output name="hypotheticals_faa" value="TEST_1/TEST_1.hypotheticals.faa"/>
            <output name="summary_txt" value="TEST_1/TEST_1.txt" lines_diff="4"/>
            <output name="annotation_plot" value="TEST_1/TEST_1_plot.svg" ftype="svg" compare="sim_size"/>
-          <output name="annotation_json" value="TEST_1/TEST_1.json" lines_diff="6"/>
-          <output name="logfile" value="TEST_1/TEST_1.log" lines_diff="15"/>
+           <output name="annotation_json" value="TEST_1/TEST_1.json" lines_diff="6"/>
+           <output name="logfile" ftype="txt" lines_diff="2">
```
Comment (bernt-matthias) [inline tools/bakta/bakta.xml:236]:
```suggestion
           <output name="logfile" ftype="txt">
```

File: tools/bakta/bakta.xml [hunk 5]
```diff
@@ -316,7 +326,16 @@
              <section name="output_files">
                  <param name="output_selection" value="log_txt,sum_txt"/>
              </section>
-             <output name="logfile" value="TEST_5/TEST_5.log" lines_diff="13"/>
+             <output name="logfile" ftype="txt" lines_diff="2">
```
Comment (bernt-matthias) [inline tools/bakta/bakta.xml:329]:
```suggestion
             <output name="logfile" ftype="txt">
```

File: tools/bakta/bakta.xml [hunk 6]
```diff
@@ -343,7 +362,16 @@
             <output name="summary_txt" value="TEST_6/TEST_6.txt" lines_diff="4"/>
             <output name="annotation_plot" value="TEST_6/TEST_6_plot.svg" ftype="svg" compare="sim_size"/>
            <output name="annotation_json" value="TEST_6/TEST_6.json" lines_diff="6"/>
-           <output name="logfile" value="TEST_6/TEST_6.log" lines_diff="15"/>
+           <output name="logfile" lines_diff="2">
```
Comment (bernt-matthias) [inline tools/bakta/bakta.xml:365]:
```suggestion
           <output name="logfile" ftype="txt">
```

File: tools/bakta/test-data/TEST_1/TEST_1.embl [hunk 7]
```diff
@@ -8,13 +8,13 @@ OS   .
 OC   .
 XX
 CC   Annotated with Bakta
-CC   Software: v1.8.1
+CC   Software: v1.8.2
```
Comment (bernt-matthias) [inline tools/bakta/test-data/TEST_1/TEST_1.embl:11]: Changes due to 

- version
- dates 
- paths

should be handled by `lines_diff`. So an update of **most** of the test files seems not really necessary, or?

File: tools/bakta/test-data/TEST_1/TEST_1.log [hunk 8]
```diff
@@ -27,7 +27,7 @@ predict ncRNA regions...
 predict CRISPR arrays...
 	found: 0
 predict & annotate CDSs...
-	predicted: 2 
+	predicted: 2
```
Comment (bernt-matthias) [inline tools/bakta/test-data/TEST_1/TEST_1.log:30]: Here the  update of the test data is fine, because the change of the format.

**Comment by pimarin** [comment]
it's find to me, you keep the past test file to validate the assert test ?

**Review by bgruening — APPROVED** [review]

**Pull request merged**